src/register_manager.cc

```diff
@@ -102,4 +102,14 @@ CandidateList RegisterManager::complete_register_name(StringView prefix, ByteCou
     return complete(prefix, cursor_pos, reg_names | transform([](auto& i) { return i.key; }) | gather<Vector<String>>());
 }
 
+RegisterMap::const_iterator RegisterManager::begin() const
```

Comment:
those could be directly inlined in the header as `auto begin() const { return m_registers.begin(); }`.

Reply:
I didn't know what would be more preferable (e.g. `auto` vs explicit type); I'll do as you suggest.